hal/inc/concurrent_hal.h

```diff
@@ -132,4 +181,3 @@ void os_thread_scheduling(bool enabled, void* reserved);
 #endif
```

Comment:
The reserved argument of os_thread_scheduling should have a default value of NULL